packages/packages/core/editor-canvas/src/mcp/tools/configure-element/prompt.ts

```diff
@@ -26,10 +29,40 @@ When a user requires to change anything in an element, such as updating text, co
 This tool handles elements of type "widget".
 This tool handles styling elements, using the "stylePropertiesToChange" parameter.
 
+# CRITICAL: Color Property Handling - MANDATORY WORKFLOW (ENFORCED - CANNOT BYPASS)
```

Comment:
I am not sure how this. I haven't much experience with prompting. Originally Angie ignored this instruction, and therefore I have enforced it strictly.

Possibly we can lessen this requirement later on, once Angie is better trained with our data.